.github/workflows/runtime-sync.yml

```diff
@@ -0,0 +1,84 @@
+name: AspNetCore-Runtime Code Sync
+on:
+  # Test this script using on: push
+  # push
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule
+    # Once per day at midnight PST (8 UTC)
+    - cron: '0 8 * * *'
+
+jobs:
+  compare_repos:
+    name: Compare the shared code in the AspNetCore and Runtime repos and notify if they're out of sync.
+    runs-on: windows-latest
+    steps:
+    - name: Checkout aspnetcore
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork 
+        # repository: 'Tratcher/aspnetcore'
```

Comment:
Nit: I like the `Test this script using changes in a fork` comment, but leaving this here might encourage people to use your fork.

```suggestion
```

Reply:
I left these intentionally so we can know which fields to change for testing.

Reply:
I think @halter73 is implying it's fine to have the comment, just to change the repository to something else, ex: dotnet/runtime.

Reply:
The point is that you can only test actions in a personal fork. This is an example of one.